src/KiwiLogin/__test__/KiwiLogin.test.js

```diff
@@ -35,7 +35,7 @@ describe('KiwiLogin', () => {
     const Login = wrapper.instance();
     await Login.handleSignIn(e);
     expect(e.preventDefault.mock.calls).toHaveLength(1);
-    expect(Login.props.history.push.mock.calls[0][0]).toBe('/faq/');
+    expect(Login.props.history.push.mock.calls).toHaveLength(0);
```

Comment:
this test shouldn't be changed, instead, `onLogin` should return object matching this flow type:

```
type Result = {|
  user_id: string,
  token: string
|}
```